views/scss/inc/_connectivity.scss

```diff
@@ -0,0 +1,23 @@
+.connectivity-box {
+    display: none;
+    width: 40px;
+    margin-left: 40px;
+    .icon-connect, .icon-disconnect {
+        display: none;
+        line-height: 1.6;
+        font-size: 2.3rem;
```

Comment:
you should use the font mixin `@include font-size(23)` to be consistent with the application.